pkg/loader/compose/v3.go

```diff
@@ -192,6 +192,11 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 		serviceConfig.Command = composeServiceConfig.Entrypoint
 		serviceConfig.Args = composeServiceConfig.Command
 
+		//Handling replicas
+		if composeServiceConfig.Deploy.Replicas != nil {
```

Comment:
no need to add `!= nil`. Just leave as `if composeServiceConfig.Deploy.Replicas {`